tests/search/includes/classes/test-class-queue.php

```diff
@@ -1064,6 +1065,92 @@ public function test__delete_jobs_for_index_version() {
 		);
 	}
 
+	public function stat_sampling_invalid_stat_param_data() {
+		return [
+			[ array() ],
+			[ null ],
+			[ new \stdClass() ],
+			[ 5 ],
+			[ 8.6 ],
+		];
+	}
+
+	public function stat_sampling_invalid_value_param_data() {
```

Comment:
What do you think about moving the data providers right above the tests using it - e.g. - `test__maybe_update_stat_sampling_invalid_value_param` in this case.

It is a bit nitpicking though so feel free to ignore.